alpine.Dockerfile

```diff
@@ -104,7 +104,18 @@ RUN apk update \
  && apk add --no-cache \
         ca-certificates \
         snappy-dev \
-        tini
+        tini \
```

Comment:
Same thing here.

Reply:
Added comment describing why we're doing this.